docassemble/ALAutomatedTestingTests/data/questions/al_set_up_testing.yml

```diff
@@ -0,0 +1,139 @@
+metadata:
+  title: Set up AL automated testing
+  short title: Set up AL testing
+  description: Set up AssemblyLine automated testing for your docassemble package
+  comment: Test the automated integrated testing library for AssemblyLine
+---
+features:
+  question back button: True
+---
+modules:
+  - .al_set_up_testing
+---
+objects:
+  - installer: TestInstaller
+---
+# I've done this once before. Trust me...
+mandatory: True
+code: |
+  
+  # handle errors
+  
+  intro
+  
+  # da stuff
+  installer.playground_url
+  set_da_info
+  
+  # github stuff
+  
+  # confirm
+  is_ready
+  
+  # update github
+  
+  #instructions
+  next_steps
+---
+# TODO: Add link to docs where we explain how the library makes a project, etc.
+# TODO: Add links to the actual github repo and files and folders
+# TODO: only show 'other than you' if the person is logged in
+id: intro
+question: |
+  Set up AL automated integrated testing
+subquestion: |
+  This interview will help you set up automated integrated GitHub testing for your docassemble project.
+  
+  Your answers will be encrypted end-to-end and on the server. No one other than you will have access to it. You **must** make sure your GitHub account has permission to change files in the correct repository.
+
+  :clock: This could take about 15 minutes
+  
+  We will need you to give us:
+  
+  1. A [GitHub personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) with 'repo' level permissions.
+  1. Your docassemble testing account's sign-in information. That can be your account if you want it to be.
+
+  This tool will add some files to different folders in your interview's GitHub repository:
+
+  1. `tests/features/example_test.feature` is a very simple example of a test.
+  1. `.github/workflows/run_interview_tests.yml` tells GitHub when and how to run the tests.
+  1. `package.json` tells GitHub what packages the tests need.
+  1. `.gitignore` is for developers who want to work in their local environment.
+  1. `.env-example` can help guide developers who want to work in their local environment.
+  
+  It will also add 3 [secrets](https://docs.github.com/en/actions/reference/encrypted-secrets): PLAYGROUND_EMAIL, PLAYGROUND_PASSWORD, and PLAYGROUND_ID
+
+continue button field: intro
+---
+id: server info
+question: |
+  Docassemble
+subquestion: |
+  Use the information for the account that should run the tests to answer the questions below. You can use your own account as the testing account.
+fields:
+  - Testing account's email: installer.email
+    datatype: email
+  - Testing account's password: installer.password
+    datatype: password
+  - note: "From inside the testing account, hit 'Save and Run' in the YAML file you want to test. What is the URL in the address bar?"
```

Comment:
An example might be nice here. Something like "In most cases, the URL will look something like ______"